src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/TestHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SonarQube.Client.Models;
+
+namespace SonarLint.VisualStudio.Roslyn.Suppressions.UnitTests
+{
+    internal class TestHelper
+    {
+        internal static SonarQubeIssue CreateIssue(string issueKey)
```

Comment:
By convention, we use `public` for methods in `internal` classes that we expect to be called by other classes i.e. we use `internal class - public method` rather than `internal class - internal method`.

It doesn't make any difference to the compiler, but gives a clearer indication to people reading the code that the method is part of its "public" interface i.e. usable by anyone.